src/quo/components/wallet/amount_input/view.cljs

```diff
@@ -0,0 +1,85 @@
+(ns quo.components.wallet.amount-input.view
+  (:require
+    [quo.components.buttons.button.view :as button]
+    [quo.components.markdown.text :as text]
+    [quo.components.wallet.amount-input.schema :as amount-input.schema]
+    [quo.components.wallet.amount-input.style :as style]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [schema.core :as schema]))
+
+(defn- amount-button
+  [{:keys [theme accessibility-label disabled? icon on-press]}]
+  [button/button
+   {:icon-only?          true
+    :theme               theme
+    :disabled?           disabled?
+    :type                :outline
+    :accessibility-label accessibility-label
+    :size                32
+    :on-press            on-press}
+   icon])
+
+(defn- process-amount
+  [input-value min-value max-value]
+  (let [parsed-input-value (parse-double input-value)]
+    (cond
+      (nil? parsed-input-value)  min-value
+      (>= input-value max-value) max-value
+      (<= input-value min-value) min-value
+      :else                      parsed-input-value)))
+
+(defn- view-internal
+  [{:keys [on-change-text
+           auto-focus
+           init-value
+           return-key-type
+           container-style]
+    :or   {auto-focus      false
```

Comment:
That makes sense to me. Would be nice to have it [here](https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md) as it is an important rule to follow, and from what I've seen a lot of code in the codebase is using defaulting :or when destructuring instead of or. I'm happy to contribute to the guidelines update if needed!

Reply:
Fixed in this [commit](https://github.com/status-im/status-mobile/pull/18687/commits/cb3d2465b65d4643d151af6690f7e2828e47cbc3).

Reply:
Thank you for the feedback. This guideline should definitely live up there.

Reply:
@ilmotta I've had some time and created the PR with the doc update https://github.com/status-im/status-mobile/pull/18731. Do you think it makes sense to do this?
If yes, I can make the PR Ready for Review.
I've used some of your sentences, thank you for that.

Reply:
That's very useful @nazariifenii. Looks good o/